app/src/main/java/org/mozilla/focus/biometrics/BiometricAuthenticationDialogFragment.kt

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.biometrics
+
+import android.app.DialogFragment
+import android.arch.lifecycle.LifecycleObserver
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import android.support.annotation.RequiresApi
+import android.support.v4.content.ContextCompat
+import android.support.v7.app.AppCompatDialogFragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import kotlinx.android.synthetic.main.biometric_prompt_dialog_content.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.